docs/file-based-configuration.md

```diff
@@ -172,6 +172,98 @@ propagator:
   composite_list: ${OTEL_PROPAGATORS}
 ```
 
+## Instrumentation Configuration
+
+You can configure traces, metrics, and logs instrumentations.
+For more details and updates, see: [Instrumentation list and documentation](config.md#instrumentations)
+To disable a instrumentation, comment out or remove its corresponding entry.
+
+``` yaml
+instrumentation/development:
+  dotnet:
+    traces:
+      aspnet:              # ASP.NET (.NET Framework) MVC/WebApi [Framework only]
```

Comment:
I think that you should put here a name from the config.md tables.
I would like to avoid 2 places to maintain same data.